t/balancer.t

```diff
@@ -332,10 +332,10 @@ last peer failure: failed 500
 --- response_body_like: 503 Service Temporarily Unavailable
 --- error_code: 503
 --- grep_error_log eval: qr{last peer failure: \S+ \S+}
---- grep_error_log_out
-last peer failure: nil nil
-last peer failure: failed 502
-last peer failure: failed 502
+--- grep_error_log_out eval
+qr{last peer failure: nil nil
```

Comment:
We should use `\A` and `\z` to anchor the match otherwise any unexpected leading or trailing garbage cannot be detected.